documentation/modules/source-vm-prerequisites.adoc

```diff
@@ -8,9 +8,17 @@
 
 The following prerequisites apply to all migrations:
 
-* ISO/CDROM disks must be unmounted.
-* Each NIC must contain one IPv4 and/or one IPv6 address.
-* The operating system of a VM must be certified and supported as a link:https://access.redhat.com/articles/973163#ocpvirt[guest operating system with {virt}].
+* ISO/CDROM disks are unmounted.
+* Each NIC contains one IPv4 address, one IPv6 address, or both.
```

Comment:
So the minimum requirement for each NIC is either an IPv4 address or an IPv6 address, but it can utilize both.